src/DependencyInjection/MercureExtension.php

```diff
@@ -67,9 +73,29 @@ public function load(array $configs, ContainerBuilder $container)
             $bus = $hub['bus'] ?? null;
             $attributes = null === $bus ? [] : ['bus' => $hub['bus']];
             $publisherDefinition->addTag('messenger.message_handler', $attributes);
+
+            if ($config['enable_profiler'] && class_exists(Stopwatch::class)) {
+                $container->register("$hubId.traceable", TraceablePublisher::class)
+                    ->setDecoratedService($hubId)
+                    ->setArguments([new Reference("$hubId.traceable.inner"), new Reference('debug.stopwatch')]);
+                $publishers[$name] = new Reference("$hubId.traceable");
+                if ($defaultHub === $hubId) {
+                    $container->setAlias(TraceablePublisher::class, "$hubId.traceable");
+                }
+            }
+        }
+
+        if ($config['enable_profiler']) {
+            $container->register('data_collector.mercure', MercureDataCollector::class)
```

Comment:
```suggestion
            $container->register('mercure.data_collector', MercureDataCollector::class)
```

Reply:
`data_collector.<name>` was to respect the same naming convention used by symfony/framework-bundle about DataCollectors: https://github.com/symfony/framework-bundle/blob/master/Resources/config/collectors.xml

Reply:
Yes because it's in FWBundle, but here as we aren't in FWBundle, we shouldn't "pollute" the global namespace and put every services under the `mercure.` key.